docs/docusaurus.config.js

```diff
@@ -14,6 +14,24 @@ module.exports = {
         src: "img/logo-horizontal.svg",
         srcDark: "img/logo-horizontal-dark.svg",
       },
+      links: [
+        {
+          href: "https://docs.solana.com/",
+          label: "Docs »",
+          position: "left",
+        },
+        {
+          href: "https://discordapp.com/invite/pquxPsq",
+          label: "Chat",
+          position: "right",
+        },
+
+        {
+          href: "https://github.com/solana-labs/solana",
```

Comment:
Shouldn't we link to the SPL github here?  https://github.com/solana-labs/solana-program-library